### PR TITLE
Update multi ss2 PR test inputs to confirm that the references are correct and …

### PR DESCRIPTION
**Pull request by nikellepetrillo**
…SS2 passes

### CODE REVIEW

**Comment by kbergin** [comment]
Awesome to see passing tests again!!

**Review by kishorikonwar — APPROVED** [review]
Looks good to me.

**Review by kishorikonwar — COMMENTED** [review]
The additional changes look good to me.

File: pipelines/skylab/smartseq2_multisample/mouse_paired_example.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
     "MultiSampleSmartSeq2.hisat2_ref_trans_name": "hisat2_from_rsem_star_primary_gencode_mouse_vM21",
-    "MultiSampleSmartSeq2.hisat2_ref_index": "gs://hca-dcp-analysis-pipelines-reference/alignmentReferences/GRCm38_Gencode_vM21/hisat2_primary_gencode_mouse_vM21.tar.gz",
-    "MultiSampleSmartSeq2.hisat2_ref_trans_index": "gs://hca-dcp-analysis-pipelines-reference/alignmentReferences/GRCm38_Gencode_vM21/hisat2_from_rsem_star_primary_gencode_mouse_vM21.tar.gz",
-    "MultiSampleSmartSeq2.rsem_ref_index": "gs://hca-dcp-analysis-pipelines-reference/alignmentReferences/GRCm38_Gencode_vM21/rsem_primary_gencode_mouse_vM21.tar",
+    "MultiSampleSmartSeq2.hisat2_ref_index": "gs://hca-dcp-sc-pipelines-test-data/alignmentReferences/20210309/mm10/hisat2_primary_gencode_mouse_vM21.tar.gz",
+    "MultiSampleSmartSeq2.hisat2_ref_trans_index": "gs://hca-dcp-sc-pipelines-test-data/alignmentReferences/20210309/mm10/hisat2_from_rsem_star_primary_gencode_mouse_vM21.tar.gz",
+    "MultiSampleSmartSeq2.rsem_ref_index": "gs://hca-dcp-sc-pipelines-test-data/alignmentReferences/20210309/mm10/star_primary_gencode_mouse_vM21.tar",
```
Comment (jessicaway) [inline pipelines/skylab/smartseq2_multisample/mouse_paired_example.json:9]: I assume the star and rsem references are equivalent? @kishorikonwar is that correct?

**Review by jessicaway — APPROVED** [review]

**Pull request merged**